test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -563,15 +573,79 @@ public void TestResultHandlerShouldWriteToConsoleShouldShowPassedTestsForNormalV
             loggerEvents.WaitForEventCompletion();
 
             this.mockOutput.Verify(o => o.Write(PassedTestIndicator, OutputLevel.Information), Times.Once);
-            this.mockOutput.Verify(o => o.WriteLine("TestName [1h 2m]", OutputLevel.Information), Times.Once);
+            this.mockOutput.Verify(o => o.WriteLine("TestName [1 h 2 m]", OutputLevel.Information), Times.Once);
             this.mockOutput.Verify(o => o.Write(FailedTestIndicator, OutputLevel.Information), Times.Once);
-            this.mockOutput.Verify(o => o.WriteLine("TestName [4m 5s]", OutputLevel.Information), Times.Once());
+            this.mockOutput.Verify(o => o.WriteLine("TestName [4 m 5 s]", OutputLevel.Information), Times.Once());
             this.mockOutput.Verify(o => o.Write(SkippedTestIndicator, OutputLevel.Information), Times.Exactly(3));
             this.mockOutput.Verify(o => o.WriteLine("TestName", OutputLevel.Information), Times.Exactly(3));
             this.mockProgressIndicator.Verify(pi => pi.Pause(), Times.Exactly(5));
             this.mockProgressIndicator.Verify(pi => pi.Start(), Times.Exactly(5));
         }
 
+        [DataRow(".NETFramework,version=v4.5.1", "(net451)", "quiet")]
```

Comment:
What happens when the framework looks something like Framework40 ?

Reply:
FrameworkArgumentProcessor will convert framework45 to  net45  before setting target framework in default logger parameters

Reply:
But still maker sure that you code accommodates framework values that are not .NET framework